packages/opentelemetry-instrumentation-azure-search/opentelemetry/instrumentation/azure_search/version.py

```
@@ -0,0 +1 @@
+__version__ = "0.1.0"
```

Ruff (0.15.17)

[warning] No newline at end of file

Add trailing newline

(W292)